src/interface_py/Makefile

```diff
@@ -70,6 +70,8 @@ install:
 #	python3 setup.py sdist bdist_wheel
 
 clean:
+	rm -f h2o4gpu/__init__.py
```

Comment:
Are you sure here?

Reply:
yes, this is because how we bundle other libs with ours (sklearn, xgboost etc) - this file gets generated during make and if I clean the dependencies then I cannot load the module because the old init.py still refers to them